README.md

```diff
@@ -1,48 +1,68 @@
 
-1. Install a language server for a language of your choice
-2. Run `LSP: Enable Language Server` from the Command Palette to enable your server.
-  * A configuration can be added under `clients` in `Preferences: LSP Settings`
-3. Open a document supported by this language server.
-4. LSP should report the language server starting in the status bar.
+Documentation is available at [LSP.readthedocs.io](https://LSP.readthedocs.io).
 
-Documentation is available at [LSP.readthedocs.io](https://LSP.readthedocs.io) or [in the docs directory](https://github.com/tomv564/LSP/blob/master/docs/index.md)
 
-## Troubleshooting
+## Available Languages <a name="available_languages"></a>
+* [Bash](https://lsp.readthedocs.io/en/latest/#bash)
+* [C/C++](https://lsp.readthedocs.io/en/latest/#clangd)
+* [CSS/LESS/SASS (SCSS only)](https://lsp.readthedocs.io/en/latest/#css)
+* [Dart](https://lsp.readthedocs.io/en/latest/#dart)
+* [Flow (JavaScript)](https://lsp.readthedocs.io/en/latest/#flow)
+* [Go](https://lsp.readthedocs.io/en/latest/#go)
+* [HTML](https://lsp.readthedocs.io/en/latest/#html)
+* [Java (Eclipse)](https://lsp.readthedocs.io/en/latest/#java)
+* [Java (IntelliJ)](https://lsp.readthedocs.io/en/latest/#intellij)
+* [JavaScript/TypeScript](https://lsp.readthedocs.io/en/latest/#typescript)
+* [JSON](https://lsp.readthedocs.io/en/latest/#json)
+* [Julia](https://lsp.readthedocs.io/en/latest/#julia)
+* [Kotlin](https://lsp.readthedocs.io/en/latest/#kotlin)
+* [PHP (Felix Becker)](https://lsp.readthedocs.io/en/latest/#php)
+* [PHP (Intelephense)](https://lsp.readthedocs.io/en/latest/#intelephense)
+* [Polymer](https://lsp.readthedocs.io/en/latest/#polymer)
+* [Python](https://lsp.readthedocs.io/en/latest/#python)
+* [Python (Microsoft)](https://lsp.readthedocs.io/en/latest/#python_microsoft)
+* [R](https://lsp.readthedocs.io/en/latest/#r)
+* [Reason](https://lsp.readthedocs.io/en/latest/#reason)
+* [Ruby](https://lsp.readthedocs.io/en/latest/#ruby)
+* [Rust](https://lsp.readthedocs.io/en/latest/#rust)
+* [Scala](https://lsp.readthedocs.io/en/latest/#scala)
+* [Vue (JavaScript)](https://lsp.readthedocs.io/en/latest/#vue)
 
-Enable the `log_debug` setting, restart Sublime and open the console.
-See the [Troubleshooting](https://lsp.readthedocs.io/en/latest/#troubleshooting) guide for tips and known limitations.
+See [Language Server Protocol](https://microsoft.github.io/language-server-protocol/implementors/servers/) for more available implementations. Please create issues/pull requests so we can get support for more languages.
 
-Have you added multiple folders to your Sublime workspace? LSP may not handle your second folder as expected, see [this issue](https://github.com/tomv564/LSP/issues/33) for more details.
+## Troubleshooting Issues
```

Comment:
Maybe rename this to Getting help?

The section covers both troubleshooting individually and in gitter, we could suggest people to look at/create issues as well.